charts/controller/templates/statefulset.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   {{ template "iter8-controller.labels" . }}
 spec:
   serviceName: {{ .Release.Name }}
+  replicas: {{ default 1 .Values.replicas | int }}
```

Comment:
Is it strange to provide a default in `values.yaml` as well as a default in the charts? I suppose redundancy does not hurt.